src/flisp/table.c

```diff
@@ -89,16 +87,16 @@ value_t fl_table(value_t *args, uint32_t nargs)
 {
     size_t cnt = (size_t)nargs;
     if (cnt & 1)
-        lerror(ArgError, "table: arguments must come in pairs");
+        lerror(fl_ArgError, "table: arguments must come in pairs");
     value_t nt;
     // prevent small tables from being added to finalizer list
     if (cnt <= HT_N_INLINE) {
-        tabletype->vtable->finalize = NULL;
-        nt = cvalue(tabletype, sizeof(htable_t));
-        tabletype->vtable->finalize = free_htable;
```

Comment:
And this should be the only change that is not simply renaming so that I can make the `vtable` field (and the `vtab` argument) a const pointer. @JeffBezanson